web-scraping/readme.md

```diff
@@ -0,0 +1,146 @@
+> **The goal of this guide:** You will learn how to extract information from certain web pages using [developer tools](guides/fixing-errors/readme.md), [node.js](http://blog.modulus.io/absolute-beginners-guide-to-nodejs) and [X-Ray](https://www.npmjs.com/package/x-ray).
+
+# Web scraping with [node.js](http://blog.modulus.io/absolute-beginners-guide-to-nodejs) an [x-ray](https://www.npmjs.com/package/x-ray)
+
+You can of course start here at [hackertyper](http://hackertyper.com/) and do your [swordfish](http://www.imdb.com/title/tt0244244/?ref_=nv_sr_1) magic! 🤓
+
+Web scraping enables you to extract information from the web and put it in a more workable format (*.json*, *.csv*, *excel*, *google spreadsheets*).     
+
+## First things first
+
+1. Create a file `'filename'.js` in your designated folder. In this file you will write your code.
+
+* Open the [Terminal](http://www.macworld.co.uk/feature/mac-software/get-more-out-of-os-x-terminal-3608274/) application on your mac, go to the folder where you created your files by typing `cd 'path to your folder'` enter.
+
+* Install node.js: `npm install node` and enter. Then you need X-Ray: `npm install x-ray` enter.
+
+## Lets start scraping
+
+In this example we extract the 'experience' information from a certain LinkedIn profile. Important is that you don't log in! Or else the html code is different from this example.
+
+### Open Chromes Developer tools
+
+In the Chrome web browser open the devtools with `cmd-alt-i` and click on the parent element:
+
+<img src="../images/devtools-web-scraping.png" width="800">
+
+> With scraping you need to get comfortable with developer tools and with reading html code.
+
+We search 🕵 from top to bottom to find what we want:
+
+* We want all the information of experience. `class='positions'`
+* We want the title of each experience: 'Content developer'. `class='item-title'`.
```

Comment:
'Content developer'  not entirely clear that it is an example.

Perhaps add e.g.